x/incentives/types/msg.go

```diff
@@ -0,0 +1,32 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	errortypes "github.com/cosmos/cosmos-sdk/types/errors"
```

Comment:
refactor, use the new errors pkg
```suggestion
	errorsmod "cosmossdk.io/errors"
```